README.md

```diff
@@ -12,8 +12,15 @@ If you're using CAP and have an option to file tickets using the [SAP Service Ma
 
 If you don't have that option, you can use the [Issues](https://github.com/sapmentors/cap-community/issues) in this repository (please check to see that you're not creating a duplicate first). 
 
-Please [post pure questions](https://answers.sap.com/questions/ask.html?primaryTagId=9f13aee1-834c-4105-8e43-ee442775e5ce), appropriately tagged (this link will do that for you), in the SAP Community Q&A.
+Please [post pure questions](https://answers.sap.com/questions/ask.html?primaryTagId=9f13aee1-834c-4105-8e43-ee442775e5ce), appropriately tagged (this link will do that for you), in the SAP Community Q&A. Your questions appear then on the SAP Community [CAP Landing Page](https://community.sap.com/topics/cloud-application-programming).
 
 For interactive discussions register for the [SAP Mentors and Friends Slack Workspace](https://sapmentors-slack-invite.cfapps.eu10.hana.ondemand.com/) and join the [#CAP](https://sapmentors.slack.com/messages/CNX247875) channel.
 
 On social media such as Twitter, please use the `#CAP` hashtag. Thanks, and welcome!
+
+## Resources
+
+* [Official Documentaiton](https://cap.cloud.sap/docs/)
+* [SAP Community Landing Page](https://community.sap.com/topics/cloud-application-programming)
```

Comment:
perhaps should be "CAP Landing Page" as used earlier?

Reply:
Fixed in 5ba102b

Reply:
s/Documentaiton/Documentation/

I'd do it myself here but I can't figure out how I could do that without direct access ...